pkg/interceptors/interceptors_test.go

```diff
@@ -240,7 +243,7 @@ func TestGetInterceptorParams_Error(t *testing.T) {
 	}
 }
 
-func Test_GetSecretToken(t *testing.T) {
+func TestGetSecretToken(t *testing.T) {
 	tests := []struct {
 		name   string
 		cache  map[string]interface{}
```

Comment:
It's a bit confusing from the diff, but I suspect that `TestResolvePath` which is the function for the changes below should be renamed to `TestGetName` ?

Reply:
yup good catch, changed the name and made it part of TriggerInterceptor.